src/components/PeopleTable.tsx

```diff
@@ -1,645 +1,292 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+import { Link, useLocation, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type People = {
+  name: string;
+  sex: string;
+  born: number;
+  died: number;
+  motherName: string | null;
+  fatherName: string | null;
+  slug: string;
+};
+
+type SortByWhat = 'sex' | 'born' | 'died' | 'name';
+
 export const PeopleTable = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<People[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const { pathname } = useLocation();
+  const [filteredPeople, setFilteredPeople] = useState<People[]>([]);
+  const [currentIsReverse, setCurrentIsReverse] = useState<boolean>();
+  const [currentSortParam, setCurrentSortParam] = useState('');
+  const [errorText, setErrorTest] = useState('');
```

Comment:
There's a typo in the state variable name `setErrorTest`. It should be `setErrorText` to match its intended purpose.